.github/workflows/ci_tier2.yaml

```diff
@@ -119,6 +119,9 @@ jobs:
       - name: Build CLI
         run: make build-cli
 
+      - name: Build minimal cli
```

Comment:
why do we need to build two different version of CLI here?

Reply:
Maybe this isn't the right place to put it but I'd like the minimal build or a check in CI. The `consumer` feature flag for the cli is needed for the build to work so I'd like to avoid that for this feature flag.